src/components/StillHaveQuestion/StillHaveQuestion.tsx

```diff
@@ -18,8 +18,12 @@ const StillHaveQuestion: React.FC = () => {
           {intl('frequently.asked.questions.still.have.questions.description')}
         </SupportChat>
       </HaveQuestion>
-      <ButtonLink type="primary" to="/contact-us">
-        Contactanos <TbSend />
+      <ButtonLink
+        type="primary"
+        to="/#contact-us"
```

Comment:
This should redirect to the Contact Us page, not the section.

Reply:
Done!